src/app/state/accountHistorySlice.ts

```diff
@@ -65,6 +69,51 @@ export const fetchAccountHistory = createAsyncThunk<
   }
 });
 
+export const fetchAccountHistoryAllPairs = createAsyncThunk<
+  SdkResult,
```

Comment:
You can simply return `adex.OrderReceipt[]` here, it will simplify things.
```suggestion
  adex.OrderReceipt[],
```